### PR TITLE
Fix assert on the acutual http code

### PR DESCRIPTION
**Pull request by atsushi-ishibashi**
Issue #524

Description of changes:
Replace `is_good_response` with functions checking each status code.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by andreeaflorescu** [comment]
Please sign your commit:
```bash
git commit --amend -s --no-edit
git push -f
```

File: tests/integration_tests/functional/test_mmds.py [hunk 1]
```diff
@@ -28,30 +28,6 @@ def test_mmds(test_microvm_with_ssh, network_config):
     assert test_microvm.api_session.is_status_ok(response.status_code)
     assert response.json() == {}
 
-    # Test that patch return NotFound when the MMDS is not initialized.
```
Comment (andreeaflorescu) [inline tests/integration_tests/functional/test_mmds.py]: I like it better with one test/resource. Is one of the python linters complain about too many statements?

Comment (atsushi-ishibashi) [reply tests/integration_tests/functional/test_mmds.py]: Yes, it complained that the number of statements was over 50.

Comment (andreeaflorescu) [reply tests/integration_tests/functional/test_mmds.py]: In this case you will need to squash this commit and the following 2 into one. The next two commits are there to fix this one from what I understand.

Comment (andreeaflorescu) [reply tests/integration_tests/functional/test_mmds.py]: Also, you have a typo in the commit message (statesments).

Comment (atsushi-ishibashi) [reply tests/integration_tests/functional/test_mmds.py]: Thanks! I squashed and fix typo.

File: tests/framework/http.py [hunk 2]
```diff
@@ -22,7 +22,27 @@ def is_good_response(response: int):
             """Return `True` for all HTTP 2xx response codes."""
             return 200 <= response < 300
 
+        def is_status_ok(response: int):
```
Comment (andreeaflorescu) [inline tests/framework/http.py]: nit: you have a typo in this commit message as well. Sorry, I didn't notice (/s/actutual/actual).

Comment (atsushi-ishibashi) [reply tests/framework/http.py]: Oh, sorry. Thanks!

**Pull request merged**